### DISCUSSION TITLE
vigilant mode enabled, no 'unverified' badge for commits I didn't author

### DISCUSSION THREAD

**Discussion by fredemmott** [community/community#27332] [Other Feature Feedback, Questions, & Ideas]
For example, I did not author/commit/work on the automation for https://github.com/hhvm/hhast/commit/6f77d97aefc15cf602bb0e0d0c7454c86d2084ce

I would expect:

1) an 'unverified' badge on that commit
2) if for some reason it is not considered unverified, a verified or partially-verified badge with details on hover

I would also expect this on the pull requests https://github.com/hhvm/hhast/pull/493 and https://github.com/hhvm/hhast/pull/492

**Comment by jfullstackdev** [comment]
I see also that you are a regular contributor to that project, any way that your PAT is shared accidentally ? or is it related to GitHub Actions ? I've seen one post, the commits came from GitHub Actions but not very sure if that is coming from that really

**Reply by fredemmott** [reply]
I have no PATs, and they no longer have any authorized apps.

Under my previous employer (who owns that project), I added a GitHub actions configuration which merely ran tests, never made commits. after I left that company, an employee extended it, adding these commits

Regardless, even if there was authorization, there should be a verified/partially verified/unverified badge with details

**Reply by jfullstackdev** [reply]
yup, that commit is unverified

how about this, did you notice this 

![image](https://user-images.githubusercontent.com/47092464/181666539-951b0a10-f16d-4d07-a1b3-3c81e329c410.png)

hopefully one from the Staff will notice this irregularities

or maybe you can try requesting it here 

https://support.github.com/contact/account

**Comment by leomotors** [comment]
Vigilant mode starting to act weird recently. Glad that there is someone who also notices this.

Normal behavior is, for any commit that is not signed (no matter who commits) should be marked as `Unverified`.
But recently, around a few weeks ago I guess, it stopped showing `Unverified` and I have no idea why.
However, it still works for some cases like co-authored commit that is signed by other people (see: https://github.com/Leomotors/blog/commits/main) which will be shown as `Partially Verified`

I guess there are some changes or maybe bugs happening on the GitHub side and I am also looking for a fix or some announcement from them.

*Update: I submitted a bug report to support, hopefully will get some useful information*

**Reply by oliviertassinari** [reply]
I have notice the exact same thing.

**Reply by leomotors** [reply]
Update: Got a reply from support and they mention that they are aware of this issue but do not have ETA for it yet. Surely it will get fixed soon.
